core/templates/dev/head/pages/exploration_player/FeedbackPopupDirective.js

```diff
@@ -108,8 +108,12 @@ oppia.directive('feedbackPopup', [
 
           $scope.saveFeedback = function() {
             if ($scope.feedbackText) {
+              var feedbackSubject = $scope.feedbackText.split(' ', 7).join(' ');
+              if (!feedbackSubject.includes(' ')) {
```

Comment:
Not a fan of this check since languages such as Chinese do utilize spaces (but yet there are no spacing between characters) so it would result into the text being too long. Another strategy here would be to cut on character length first then by word.

Reply:
Done